examples/cpp_examples/hello-embed.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+This file is an example embedding for the Interpreter Manager class
+*/
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <iostream>
+
+#include "interpreterManager.h"
+
+mp_obj_t execute_from_str(const char *str) {
+    nlr_buf_t nlr;
+    if (nlr_push(&nlr) == 0) {
+        qstr src_name = 1/*MP_QSTR_*/;
+        mp_lexer_t *lex = mp_lexer_new_from_str_len(src_name, str, strlen(str), false);
+        mp_parse_tree_t pt = mp_parse(lex, MP_PARSE_FILE_INPUT);
+        mp_obj_t module_fun = mp_compile(&pt, src_name, false);
+        mp_call_function_0(module_fun);
+        nlr_pop();
+        return 0;
+    } else {
+        // uncaught exception
+        return (mp_obj_t)nlr.ret_val;
+    }
+}
+
+
+int main() {
+
+    Interpreter m;
+
+    const char str[] = "print('Hello world of easy embedding!')";
+    if (execute_from_str(str)) {
+        std::cout << "Error\n";
```

Comment:
Did this compile? you would need to add std::end too at the end `std::cout << "Error\n" << std::end;`